features/step_definitions/resources.rb

```diff
@@ -174,7 +174,7 @@
     res = user.cli_exec(:get, **opts)
     case res[:response]
     # the resource has terminated which means we are done waiting.
-    when /cannot get project/, /not found/, /No resources found/
+    when /Error from server (Forbidden): projects.project.openshift.io/, /is forbidden/
```

Comment:
if we run this step as cluster admin, then it will likely return `not found` or `No resources found` or something. Better to still cover that use case. Also this step is used not only for projects but other other objects. So maybe only `cannot get project` needs to be changed to `Error from server (Forbidden): projects.project.openshift.io` and the others kept intact.

Reply:
Updated as comments.